src/mediapipe_internal/mediapipegraphdefinition.cpp

```diff
@@ -461,7 +461,7 @@ Status MediapipeGraphDefinition::initializeNodes() {
                 return StatusCode::LLM_NODE_NAME_ALREADY_EXISTS;
             }
 
-            std::shared_ptr<LLMNodeResources> nodeResources = nullptr;
+            std::shared_ptr<LLMNodeResources> nodeResources = std::make_shared<LLMNodeResources>();
```

Comment:
Why are you making that change?

Reply:
The createLLMNodeResources caller needs to be able to initialize empty resources object on its own, to be able to select which class it should be. In core OVMS it is LLMNodeResources, but in unit tests where loading model is not important, it is MockedLLMNodeResources. There is no possibility to mock when you hide creation inside createLLMNodeResources.